documentation/modules/auxiliary/scanner/smb/smb_login.md

```diff
@@ -115,3 +115,12 @@ set SMBPass [password]
 
 Note: If an account has been successfully brute-forced, that account will not be tried again.
 
+Additionally, if you wish to disable automatic detection of all-access systems, you can change the following option:
```

Comment:
While this option doesn't necessarily belong to the credential format options above, it's worth mentioning as a footnote.